index.js

```diff
@@ -53,9 +62,10 @@ app.get('/favicon.ico', (req, res) => {
 });
 
 app.all('/graphql', (req, res) => res.redirect('/'));
+auth(app);
```

Comment:
So this only adds an authentication requirement for the GraphiQL routes, not for a plain GraphQL query?